src/catalog/src/builtin.rs

```diff
@@ -8131,6 +8191,24 @@ pub const MZ_ANALYTICS_CLUSTER: BuiltinCluster = BuiltinCluster {
     replication_factor: 0,
 };
 
+pub const MZ_DEFAULT_NETWORK_POLICY: BuiltinNetworkPolicy = BuiltinNetworkPolicy {
```

Comment:
Builtin Network Policies seem cool, and the implementation is good! But I'm not totally sure I understand their purpose. This seems to overlap a decent amount with the `default_network_policy_allow_list`? I scrolled through the design doc again and I didn't see them mentioned there